docs/DeveloperGuide.md

```diff
@@ -934,16 +1057,16 @@ Guarantees: The specified interview will be deleted from the applicant.
   (e.g. a `.txt` file)
 * **Graphical User Interface (GUI)**: A type of user interface that allows users to interact with software through
   graphical icons and visual indicators.
-* **UI**:
-* 
+* **UI**: 
+*
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## **Appendix: Instructions for manual testing**
 
 Given below are instructions to test the app manually.
 
-<box type="info" seamless>
+<box type="info" seamless />
```

Comment:
i think this box doesnt contain any info, is it supposed to contain the next lines note?

<img width="992" alt="image" src="https://github.com/AY2324S1-CS2103T-W08-1/tp/assets/84584280/cba61716-374e-4e64-90ea-fa3779b2524a">

Reply:
might be due to the extra line in between, maybe you can try deleting that line

Reply:
Hi ivan, thanks for the review! I've implemented all the suggestions, does it look correct now?